lms/static/styles/components/_LMSGrader.scss

```diff
@@ -26,6 +26,14 @@
 .LMSGrader__student-count {
   margin: 10px;
   font-weight: 500;
+  /**
+   * Provide a fixed width to prevent this element
```

Comment:
This comment might benefit from a well-placed comment or two, as it's a bit breathless :)